### ISSUE TITLE
Question about tft model

### ISSUE THREAD

**Issue by KeYunLee** [google-research/google-research#345]
Hi, I have a question about input embedding for tft model.
At tft.model.py line 540, I have a Dense layer one to many problem.
Input of the dense layer is regular_inputs[:, 0, i:i + 1], equal [:,1], and first dim is batch
Output of the dense layer is [:,self.hidden_layer_size]
this is a one value to vector problem. Is this reasonable?

    # Static inputs
    if self._static_input_loc:
      static_inputs = [tf.keras.layers.Dense(self.hidden_layer_size)(
          regular_inputs[:, 0, i:i + 1]) for i in range(num_regular_variables)
                       if i in self._static_input_loc] \
          + [embedded_inputs[i][:, 0, :]
             for i in range(num_categorical_variables)
             if i + num_regular_variables in self._static_input_loc]
      static_inputs = tf.keras.backend.stack(static_inputs, axis=1)
